scala/lambdas/ingest-mapper/src/main/scala/uk/gov/nationalarchives/ingestmapper/MetadataService.scala

```diff
@@ -78,7 +78,8 @@ class MetadataService(s3: DAS3Client[IO], discoveryService: DiscoveryService) {
                 if potentialSeries.contains("Unknown") then IO.pure(DepartmentAndSeriesCollectionAssets(None, None))
                 else discoveryService.getDiscoveryCollectionAssets(potentialSeries)
               collectionAssets.map { assets =>
```

Comment:
Should `assets` now be `_`?

Reply:
We're still using it on line 81, I think it's clearer to keep it.

Reply:
Ah, I missed this because of the way Github wraps the text.